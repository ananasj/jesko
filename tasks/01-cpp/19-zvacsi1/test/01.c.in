51
17
60 29 99 78 50 67 95 40 56 99 87 70 64 99 88 98 37 71 2 58 93 100 84 20 2 79 85 98 84 62 93 22 64 14 33 31 84 84 37 80 63 56 85 65 91 94 14 42 65 6 3
