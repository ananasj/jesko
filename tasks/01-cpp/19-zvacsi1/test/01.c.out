77 46 116 95 67 84 112 57 73 116 104 87 81 116 105 115 54 88 19 75 110 117 101 37 19 96 102 115 101 79 110 39 81 31 50 48 101 101 54 97 80 73 102 82 108 111 31 59 82 23 20 
